src/ExadelMentorship.BusinessLogic/ExadelMentorship.BusinessLogic.csproj

```diff
@@ -4,9 +4,10 @@
     <TargetFramework>netstandard2.0</TargetFramework>
 	 <LangVersion>9</LangVersion>
   </PropertyGroup>
-
+	
   <ItemGroup>
     <PackageReference Include="FluentValidation" Version="11.0.1" />
+    <PackageReference Include="MailKit" Version="3.3.0" />
```

Comment:
You can remove it

Reply:
I can not remove it, I'm using it SmtpClient, 
If I will use 'SmtpClient' from System.Net.Mail then it does not have 'Connect, Authenticate etc'
<img width="843" alt="image" src="https://user-images.githubusercontent.com/59729680/174801919-2ba63fb9-d514-4188-9147-90bd6102dd9b.png">

Reply:
maybe it's better to stay with MailKit? :-) 
<img width="422" alt="image" src="https://user-images.githubusercontent.com/59729680/174818909-d0cdb393-dd0a-41c5-84b0-0d9c550c9d6b.png">

Reply:
done